src/coreclr/src/jit/gentree.h

```diff
@@ -3844,7 +3897,7 @@ struct GenTreeCall final : public GenTree
 #define GTF_CALL_M_NONVIRT_SAME_THIS       0x00000080 // GT_CALL -- callee "this" pointer is
                                                       // equal to caller this pointer (only for GTF_CALL_NONVIRT)
 #define GTF_CALL_M_FRAME_VAR_DEATH         0x00000100 // GT_CALL -- the compLvFrameListRoot variable dies here (last use)
-#define GTF_CALL_M_TAILCALL_VIA_HELPER     0x00000200 // GT_CALL -- call is a tail call dispatched via tail call JIT helper.
+#define GTF_CALL_M_TAILCALL_VIA_JIT_HELPER 0x00000200 // GT_CALL -- call is a tail call dispatched via tail call JIT helper.
```

Comment:
I would prefer to avoid adding too many ifdefs around this. Right now I just have the one ifdef in `IsTailCallViaJitHelper` below, and then this can be used in all configurations with the compiler hopefully taking care of eliminating the dead code for other targets. That means we do not need ifdefs in all the places that handle this case. Does that make sense?

Reply:
Sounds good to me. @CarolEidt Are you ok with not adding an ifdef here?

Reply:
Yes, that's OK with me.